### PR TITLE
librbd: cleanup logging code under librbd/io

### PR DESCRIPTION
**Pull request by runsisi**
Signed-off-by: runsisi <runsisi@zte.com.cn>

### CODE REVIEW

File: src/librbd/io/CopyupRequest.cc [hunk 1]
```diff
@@ -93,15 +94,15 @@ CopyupRequest::~CopyupRequest() {
...
   m_pending_requests.push_back(req);
 }
 
 void CopyupRequest::complete_requests(int r) {
   while (!m_pending_requests.empty()) {
     vector<ObjectRequest<> *>::iterator it = m_pending_requests.begin();
     ObjectRequest<> *req = *it;
-    ldout(m_ictx->cct, 20) << __func__ << " completing request " << req
+    ldout(m_ictx->cct, 20) << "completing request " << req
                            << dendl;
```
Comment (dillaman) [inline src/librbd/io/CopyupRequest.cc]: Nit: you can move this to the previous line

File: src/librbd/io/CopyupRequest.cc [hunk 2]
```diff
@@ -168,7 +168,7 @@ bool CopyupRequest::send_copyup() {
     // merge all pending write ops into this single RADOS op
     for (size_t i=0; i<m_pending_requests.size(); ++i) {
       ObjectRequest<> *req = m_pending_requests[i];
-      ldout(m_ictx->cct, 20) << __func__ << " add_copyup_ops " << req
+      ldout(m_ictx->cct, 20) << "add_copyup_ops " << req
                              << dendl;
```
Comment (dillaman) [inline src/librbd/io/CopyupRequest.cc]: Nit: you can move this to the previous line

File: src/librbd/io/CopyupRequest.cc [hunk 3]
```diff
@@ -145,8 +145,8 @@ bool CopyupRequest::send_copyup() {
     // actual modification.
     m_pending_copyups.inc();
 
-    ldout(m_ictx->cct, 20) << __func__ << " " << this << " copyup with "
-                           << "empty snapshot context" << dendl;
+    ldout(m_ictx->cct, 20) << "copyup with empty snapshot context"
+                           << dendl;
```
Comment (dillaman) [inline src/librbd/io/CopyupRequest.cc]: Nit: you can move this to the previous line(?)

File: src/librbd/io/ImageRequest.cc [hunk 4]
```diff
@@ -65,16 +65,15 @@ struct C_FlushJournalCommit : public Context {
...
                    << "safe" << dendl;
 
     aio_comp->add_request();
   }
 
   void finish(int r) override {
     CephContext *cct = image_ctx.cct;
-    ldout(cct, 20) << this << " C_FlushJournalCommit: journal committed"
+    ldout(cct, 20) << "C_FlushJournalCommit: journal committed"
                    << dendl;
```
Comment (dillaman) [inline src/librbd/io/ImageRequest.cc]: Nit: you can move this to the previous line(?)

File: src/librbd/io/ObjectRequest.cc [hunk 5]
```diff
@@ -304,6 +304,9 @@ template <typename I>
 void ObjectReadRequest<I>::send_copyup()
 {
   ImageCtx *image_ctx = this->m_ictx;
+  ldout(image_ctx->cct, 20) << "send_copyup " << this << " " << this->m_oid << " "
```
Comment (dillaman) [inline src/librbd/io/ObjectRequest.cc]: This will now have "send_copyup" and ```this``` in the log message twice

**Comment by runsisi** [comment]
@dillaman updated, thanks!

**Pull request merged**